### DISCUSSION TITLE
Why does the :latest version give me v0.12.0-beta.1?

### DISCUSSION THREAD

**Discussion by kosymodo** [alexjustesen/speedtest-tracker#749] [Q&A] [answered]
Just updated my instance today, and although my Portainer container settings are to pull the :latest version, this time it has pulled v0.12.0-beta.1  Any ideas why?  The latest release should be v0.11.22 right?

![image](https://github.com/alexjustesen/speedtest-tracker/assets/66123133/82f503aa-4cdf-4d6f-9680-1b0441fc2586)

![image](https://github.com/alexjustesen/speedtest-tracker/assets/66123133/33ccff8f-3702-4c6c-801d-444ab647ef53)

I'm not too bothered that it's a beta version, but just want to understand why it pulled this version, not v.0.11.22

**Comment by alexjustesen** [comment, marked as answer]
Pipeline was misconfigured for beta releases so it got tagged as latest unfortunately. Was fixed yesterday but I can't untag it.

**Comment by kosymodo** [comment]
Ah OK that explains it Alex!  Thanks :)
